src/main/java/Duke.java

```diff
@@ -1,10 +1,62 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        // Printing the logo and greeting message
+        String logo = "  _________   _____      _____   \n" +
+                " /   _____/  /  _  \\    /     \\  \n" +
+                " \\_____  \\  /  /_\\  \\  /  \\ /  \\ \n" +
+                " /        \\/    |    \\/    Y    \\\n" +
+                "/_______  /\\____|__  /\\____|__  /\n" +
+                "        \\/         \\/         \\/ ";
+        System.out.println("Hello! I'm SAM\n" + logo + "\n" + "What can I do for you?\n");
+
+        // Initializing scanner to read user input
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine();
+
+        // Array to store tasks
+        Task[] records = new Task[100];
+        int numItems = 0; // Number of tasks
+        int listIndex; // Index for tasks
+
+        // Main loop
+        while (!line.equals("bye")) {
+            String[] words = line.split(" ");
```

Comment:
You may need to incorporate exception handling because you are taking user input